src/sentry/conf/server.py

```diff
@@ -409,6 +409,7 @@ def env(
     "django.contrib.staticfiles",
     "sentry.issues.apps.Config",
     "sentry.feedback",
+    "sentry.hybridcloud",
```

Comment:
Seems like a good longer term home for the service libraries, outboxes and RPC infrastructure as well. I like the lack of `_` in the name.